sonar-plugin-api/src/main/java/org/sonar/api/CoreProperties.java

```diff
@@ -488,7 +488,9 @@
 
   /**
    * @since 5.1
+   * @deprecated since 6.3. No longer taken into consideration as all files are always imported.
    */
+  @Deprecated
   String IMPORT_UNKNOWN_FILES_KEY = "sonar.import_unknown_files";
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)